analytic_engine/src/table/metrics.rs

```diff
@@ -93,6 +100,15 @@ lazy_static! {
         &["table"],
         exponential_buckets(0.01, 2.0, 13).unwrap()
     ).unwrap();
+
+    // Buckets: 0.0005, 0.001, 0.002, .., 262.14, +Inf
+    pub static ref PARTITION_TABLE_DURATION_HISTOGRAM: HistogramVec = register_histogram_vec!(
```

Comment:
How about defining two histograms for read and write? After that, the number of the labels of the histogram is limited to 2 rather than 3.